docs/_source/tutorials/notebooks/training-llm-mistral-sft.ipynb

```diff
@@ -0,0 +1,602 @@
+    "\n",
+    "To fit this in our `FeedbackDataset`, we have to split this up into a \"background\", a \"prompt\" with all chat history and the last prompt, and the final \"response\"."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from typing import Dict, Any\n",
+    "\n",
+    "def extract_background_prompt_response(text: str) -> Dict[str, Any]:\n",
+    "    '''Extract the anthropic prompt from a prompt and response pair.'''\n",
+    "    start_prompt = text.find(\"<human>:\")\n",
+    "    end_prompt = text.rfind(\"<bot>:\")\n",
+    "    # Background is anything before the first <human>:\n",
+    "    background = text[:start_prompt].strip()\n",
+    "    # Prompt is anything between the first <human>: (inclusive) and the last <bot>: (exclusive)\n",
+    "    prompt = text[start_prompt: end_prompt].strip()\n",
+    "    # Response is everything after the last <bot>: (inclusive)\n",
+    "    response = text[end_prompt:].strip()\n",
+    "    return {\"background\": background, \"prompt\": prompt, \"response\": response}\n",
+    "\n",
+    "\n",
+    "data = data.map(extract_background_prompt_response, input_columns=\"text\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we can trivially convert this dataset into `FeedbackRecord` instances, and add them to the `dataset`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "records = [\n",
+    "    rg.FeedbackRecord(\n",
+    "        fields={\n",
+    "            \"background\": sample[\"background\"],\n",
+    "            \"prompt\": sample[\"prompt\"],\n",
+    "            \"response\": sample[\"response\"],\n",
+    "        },\n",
+    "    )\n",
+    "    for sample in data\n",
+    "]\n",
+    "dataset.add_records(records)\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now that we have all the records locally, we want to push them to the Argilla server as well. Only then will we be able to see them in the Argilla UI."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "dataset.push_to_argilla(\"oig-30k\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "![training-llm-mistral-sft](../../_static/tutorials/training-llm-mistral-sft/training-llm-mistral-sft.png)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## (Optional) Annotate instruction samples\n",
+    "If you are using your own proprietary data, data from an unreliable source, or require on your annotators to provide responses, then you must perform data annotation. However, for the purposes of this tutorial, we will assume that all data is high quality and skip this step."
```

Comment:
maybe add a reference to the docs/video from Natalia here?

Reply:
Which reference do you think is best?
https://docs.argilla.io/en/latest/practical_guides/annotate_dataset.html perhaps?

Reply:
Yes that would be good but maybe https://docs.argilla.io/en/latest/conceptual_guides/llm/llm.html is better from a conceptual point of view?

Reply:
Sounds good, solved in c53ac20136bfdf6c5ecda13067b4dbab25e4c970